src/css/Support.css

```diff
@@ -0,0 +1,32 @@
+.Support-Text{
+    color: black;
+    text-align: left;
+    opacity: 1;
+    font: Regular 25vh/30vh Whyte Inktrap;
+    letter-spacing: 0;
+    -webkit-transform: translate(-50%, -40%);
+    transform: translate(-50%,-40%);
+    position: absolute;
+    top: 50%;
+    left:30%;
+    white-space: pre-line;
+    height: 50%;
+    width: 25%;
+  }
+  
+.Header-Text{
+    text-align: left;
+    font-family: Medium 127vh/120vh Whyte Inktrap;
+    font-size: 15vh;
```

Comment:
Change the font size to 13.5vh so that it resembles 127px from the ref. 15vh is a little too big. Also, we are using the Lato font now.